package-lock.json

```diff
@@ -0,0 +1,3899 @@
+{
```

Comment:
We already have `yarn.lock`, so it is not needed.

Reply:
Done